sunkit_image/tests/test_granule.py

```diff
@@ -0,0 +1,276 @@
+import unittest
+import random
+import numpy as np
+import sunpy
+import sunpy.map as sm
+from sunpy.coordinates import frames
+from astropy.io import fits
+import os
+# import pathlib as pl
+# import scipy
+# import scipy.io as sio
+import shutil
+import astropy.units as u
+from astropy.coordinates import SkyCoord
+import sunkit_image.granule as granule
+
+
+class TestUtils(unittest.TestCase):

```

Comment:
While this is a valid set of unit tests.
We want to avoid using `unittest` directly and the entire class structure. 

I would prefer this is was written using in the pytest format: https://docs.pytest.org/en/7.2.x/ 
It would mean removing the class and any setup or teardown would be pytest fixtures.

I am happy to help with this.